src/components/media/Media.scss

```diff
@@ -11,14 +11,13 @@
     align-self: flex-start;
     flex: 0 0 auto;
     justify-self: flex-start;
-    margin: 0;
+    margin: 0 10px 0 0;
```

Comment:
What is the convention for this, would it make sense to move this layout styling to `.bdl-Media` instead of baking it into the `.bdl-Media-figure`?

Reply:
No, this is correct.
- If the figure is present, it should push over the content (body).
- If not present, the body is full-width.
- The specificity for overriding the margin should be (0,0,1)

Reply:
Ah that makes sense, didn't realize figure could be absent